Model/BasePoiPiaPaymentMethod.php

```diff
@@ -0,0 +1,140 @@
+<?php
+/**
+ * Shop System Plugins:
+ * - Terms of Use can be found under:
+ * https://github.com/wirecard/oxid-ee/blob/master/_TERMS_OF_USE
+ * - License can be found under:
+ * https://github.com/wirecard/oxid-ee/blob/master/LICENSE
+ */
+
+namespace Wirecard\Oxid\Model;
+
+use Wirecard\PaymentSdk\Config\Config;
+use Wirecard\PaymentSdk\Config\PaymentMethodConfig;
+use Wirecard\PaymentSdk\Transaction\PoiPiaTransaction;
+use Wirecard\PaymentSdk\Transaction\Transaction;
+
+use Wirecard\Oxid\Core\Helper;
+
+/**
+ * Base class for implementation of Payment on Invoice/Payment in Advance
+ *
+ * @since 1.3.0
+ */
+abstract class BasePoiPiaPaymentMethod extends SepaCreditTransferPaymentMethod
+{
+    /**
+     * @inheritdoc
+     *
+     * @var bool
+     *
+     * @since 1.3.0
+     */
+    protected static $_bMerchantOnly = false;
```

Comment:
not needed, is false by default

Reply:
We have to set it to false, because this class is extended from ``SepaCreditTransferPaymentMethod``, where ``$_bMerchantOnly = true``.